autogpt_platform/backend/.env.example

```diff
@@ -95,6 +95,9 @@ REPLICATE_API_KEY=
 # Ideogram
 IDEOGRAM_API_KEY=
 
+# Jina
```

Comment:
Does this work if you set it and don't on the credential?

Reply:
just tested and no this does not work with the new creds setup so i will remove this from the PR

Reply:
I’m okay with this for now if you are

Reply:
fixed in my pr